src/admin/paths/admin@projects@bulk.yaml

```diff
@@ -38,19 +38,19 @@ post:
                     type: string
                     nullable: true
                   data:
-                    type: string
+                    type: object
```

Comment:
This is the actual change. The `type` has to be object for the bulk errors and this also works fine for other errors